WBooksTests/NewBookViewModelTests.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  NewBookViewModelTests.swift
+//  WBooksTests
+//
+//  Created by Juan Silva on 25/06/2021.
+//
+
+import XCTest
+@testable import WBooks
+
+class NewBookViewModelTests: XCTestCase {
+    
+    func testAddBook() throws {
+        let bookJSON = """
+                        { "title": "Testing", "author": "Juan Silva", "genre": "Coding", "year": "2021", "image": "URLImageBook", "status": "Unavailable" }
+                       """
+        
+        let newBookViewModel = NewBookViewModel(bookRepositoty: MockBookRepository(bookJson: bookJSON))
+        let book = UnidentifiedBook(title: "Testing", author: "Juan Silva", genre: "Coding", year: "2021", image: "URLImageBook", status: Status.unavailable)
+        
+        let promise = expectation(description: "Book add")
```

Comment:
If this is an expectation, we should name the constant accordingly. For example, we could call it `bookExpectation`, if you want.